api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -352,7 +376,7 @@ private Cohort findCohort(long cohortId) {
 
     private void validateMatchingWorkspace(String workspaceNamespace, String workspaceName, long workspaceId) {
         Workspace workspace = workspaceService.getRequired(workspaceNamespace, workspaceName);
-        if (workspace.getWorkspaceId() != workspaceId) {
+        if (workspace == null || workspace.getWorkspaceId() != workspaceId) {
```

Comment:
FYI, `workspaceService.getRequired` is already throwing a `NotFoundException` if the workspace is `null`, but `workspaceService.get` does not (hence the required part of the method name I think).

Reply:
Nice catch.. i was wondering why i wasn't doing a null check for this before now. I forgot and didn't dig deep enough. Removing the null check and fixed the test case to reflect this change.